.github/workflows/plugin-check.yml

```
@@ -0,0 +1,83 @@
+          wp plugin status webchangedetector --allow-root
+
+      - name: Composer install phpcs
+        working-directory: wordpress
+        run: |
+          cd ..
+          composer install
+
+      - name: Run PHPCS on plugin
+        working-directory: wordpress
+        run: |
+          ./vendor/bin/phpcs --standard=WordPress wordpress/wp-content/plugins/webchangedetector
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)